### DISCUSSION TITLE
Easy SD card access hack

### DISCUSSION THREAD

**Discussion by Hububalli** [ElectroTechnique/electrotechnique.github.io#25] [Hardware Issues]
Just a quick FYI. In one TSYNTH, I have no access to the sdcard so tried using one of these card extensions. I cut a slot in the back of the case and glued it in. It is now super easy to try out new patches.

![image](https://user-images.githubusercontent.com/8293708/110789506-dd3df680-8267-11eb-948f-4265a60da0aa.png)

**Comment by AnalogLabSwiss** [comment]
Happy to hear that. I had the same idea but used another model with mounting holes and it didn't worked.

I have to spend more time to check what is the problem.

Thanks.

**Reply by Hububalli** [reply]
I tried looking for one like that with mounting holes. Do you have the details? 

Thanks
